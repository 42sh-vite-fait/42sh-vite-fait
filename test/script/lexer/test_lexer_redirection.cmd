2>&1 >/dev/null ls