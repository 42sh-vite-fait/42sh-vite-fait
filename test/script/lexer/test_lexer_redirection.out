TOKENS:
IO_NUMBER (start: 0, len: 1) [2]
GREATAND (start: 1, len: 2) [>&]
WORD (start: 3, len: 1) [1]
GREAT (start: 5, len: 1) [>]
WORD (start: 6, len: 9) [/dev/null]
WORD (start: 16, len: 2) [ls]
